sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -93,6 +94,8 @@ void addBreadcrumb(@NotNull Breadcrumb breadcrumb, boolean executeBeforeBreadcru
       }
 
       if (breadcrumb == null) {
+        //        options.getLogger().log(SentryLevel.INFO, "Breadcrumb was dropped by scope
```

Comment:
Sounds fair to have this log. Perhaps in Debug mode instead?
Or remove dead code?

Reply:
actually we don't have access to options object on this class, that's why, it's a TODO.
I'll try to fix this "workaround" with the Scope class and Before BreadCrumb before GA, it's a design flaw.